nss-day-cohort-48/giffygram-looksmart
# Al branch fetchcaller

#### Changes Made
1. Modified file `main,js` to include `fetchCall invocations` and functionality.
2. Refactored 'provider.js' to accord with fetch call order
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout al-branch-fetchcaller
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Check out new fetch calls on main and new order of provider.js
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
